parsec/backend/postgresql/handler.py

```diff
@@ -25,6 +25,8 @@
 
 CREATE_MIGRATION_TABLE_ID = 2
 MIGRATION_FILE_PATTERN = r"^(?P<id>\d{4})_(?P<name>\w*).sql$"
+# We use int to avoid float error
+US_BETWEEN_1970_AND_2000: int = 946684800000000
```

Comment:
```suggestion
MICRO_SECONDS_BETWEEN_1970_AND_2000: int = 946684800000000
```
I would prefer a more explicit naming for this constant